framework/web_app/src/helpers/OfferSortByOptions.ts

```diff
@@ -0,0 +1,36 @@
+import { ScrapedProductOffer } from 'src/models/ScrapedProductOffer';
+import { sortOffersByFullNameAsc, sortOffersByFullNameDesc, sortOffersByPriceHighToLow, sortOffersByPriceLowToHigh, sortOffersBySpecialBestToWorst } from './ScrapedProductOfferLogic';
+
+export interface IOfferSortByOption {
+    description: string;
+    isDefault: boolean;
+    sort: (offers: ScrapedProductOffer[]) => ScrapedProductOffer[];
+}
+
+export const OfferSortByOptions: IOfferSortByOption[] = [
+    {
+        description: 'Name (A - Z)',
+        isDefault: true,
+        sort: sortOffersByFullNameAsc
+    },
+    {
+        description: 'Name (Z - A)',
+        isDefault: false,
+        sort: sortOffersByFullNameDesc
+    },
+    {
+        description: 'Price (Low to High)',
```

Comment:
when i shop, the most essential thing to me is the unit price, see woolies has sort by "Unit Price (High to Low)" and "Unit Price (Low to High)"

these i reckon we should definitely do, although i can see how they'd be difficult as we'd have to do quite a bit of conversions

happy to make this a future feature on the excel sheet, thoughts?

Reply:
Can discuss, need to think about how to approach this. But yeah i think we push it out into a separate PR at this point so this stuff goes through.